File: SDKs/amazonaws.com/globalaccelerator/2018-08-08/python/src/sdk/models/shared/updateacceleratorattributesresponse.py
from dataclasses import dataclass, field
from datetime import date, datetime
from marshmallow import fields
import dateutil.parser
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class UpdateAcceleratorAttributesResponse:
    accelerator_attributes: Optional[AcceleratorAttributes] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('AcceleratorAttributes') }})
    

File: SDKs/amazonaws.com/globalaccelerator/2018-08-08/typescript/src/sdk/models/shared/updateacceleratorattributesresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { AcceleratorAttributes } from "./acceleratorattributes";



export class UpdateAcceleratorAttributesResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=AcceleratorAttributes" })
  acceleratorAttributes?: AcceleratorAttributes;
}
